tests/generation/FileGlobs.py

```diff
@@ -29,7 +29,7 @@ def test_get_project_files(self):
         open(os.path.join("ignore_dir", "src.c"), "w").close()
         open(os.path.join("ignore_dir", "src.js"), "w").close()
 
-        with suppress_stdout(), simulate_console_inputs("**.c", "ignore_dir/**"):
+        with suppress_stdout(), simulate_console_inputs("ignore_dir/**"):
```

Comment:
:point_up: [August 4, 2016 11:30 PM](https://gitter.im/coala-analyzer/coala?at=57a382cd1c2bf6621bbc502e) for reference